README.md

```diff
@@ -340,6 +340,7 @@ Examples exist on [codesandbox.io][examples]:
 
 - [downshift Apollo example](https://codesandbox.io/s/j2omZpK3W)
 - [downshift Spectre.css example](https://codesandbox.io/s/M89KQOBRB)
+- [downshift GenieJS example](https://codesandbox.io/s/4zNL2WW1)
```

Comment:
You need to do: `import genie from 'geniejs'`

And then use `genie`. I have an update [here](https://codesandbox.io/s/NxR9wkWzN) that's using the right import, but there seems to be another issue elsewhere.